packages/components/src/theme/tokens.ts

```diff
@@ -1,3 +1,4 @@
+/** @deprecated */
```

Comment:
```suggestion
/** @deprecated since v12.1.0 please import { variables } from import '@midas-ds/components/theme' */
```

Ska man göra något sådant för att få folk att byta system?

Reply:
Japp klart tydligare

Reply:
1adf6a5ffa51e97eabf175f8c5cb4aab211c33e6